tests/test_colocation_3d.py

```diff
@@ -6,14 +6,16 @@
 import numpy as np
 import pytest
 
+from pyaerocom.io.read_earlinet import ReadEarlinet
 from pyaerocom import GriddedData
 from pyaerocom.colocation_3d import ColocatedDataLists, colocate_vertical_profile_gridded
 from tests.fixtures.data_access import TEST_DATA
 
-ROOT = TEST_DATA["Earlinet-test-3d-collocation"].path
 
-TEST_FILE = [
-    ROOT / "earlinet_example_for_ci.pkl",
+ROOT = TEST_DATA["Earlinet-test"].path
```

Comment:
"Earlinet-test-3d-collocation" is no longer used, so please remove it from TEST_DATA